src/operator/tensor/elemwise_unary_op.cc

```diff
@@ -331,7 +331,7 @@ MXNET_OPERATOR_REGISTER_BINARY(_backward_log1p)
 
 // expm1
 MXNET_OPERATOR_REGISTER_UNARY(expm1)
-.describe(R"code(Calculate ``exp(x) - 1``
+.describe(R"code(Returns element-wise ``exp(x) - 1`` value of the input.
```

Comment:
The use of the word "value" is akward. Change to:

Returns``exp(x) - 1``, computed element-wise on the input.